src/modules/processes.nix

```diff
@@ -70,7 +70,7 @@ in
               default = true;
             };
             unix-socket = lib.mkOption {
-              type = types.str;
+              type = types.str; # TODO: should be path?
```

Comment:
Paths get copied to the /nix/store. Most of the time you don't want that, especially for devenv runtime state.

Reply:
TIL! Nice